src/harness.js

```diff
@@ -234,11 +235,41 @@ class Harness {
       /* this.log respects verbose option, use logFormErrors here */
       console.error(`Error encountered while filling form:`, JSON.stringify(fillResult.errors, null, 2));
     }
+    return fillResult;
+  }
 
+  /**
+   * Loads and fills a contact form,
+   *
+   * @param {string} contactType Type of contact that should be created
+   * @param  {...string[]} answers Provide an array for the answers given on each page. See fillForm for more details.
+   */
+  async fillContactForm(contactType, ...answers) {
+    const fillResult = await this._fillContactForm(contactType, 'create', ...answers);
     this.pushMockedDoc(...fillResult.contacts);
     return fillResult;
   }
 
+  /**
+   * Loads and fills a contact edit form,
+   *
+   * @param {string} contactType Type of contact that should be created
+   * @param  {...string[]} answers Provide an array for the answers given on each page. See fillForm for more details.
+   */
+  async fillContactEditForm(contactType, ...answers) {
+    const assignValues = (target, props) =>
+      Object.entries(props).forEach(([k, v]) => (k !== '_id' && v) && (target[k] = v)); // replace all non empty/null values
+
+    const fillResult = await this._fillContactForm(contactType, 'edit', ...answers);
+    const _contact = this.subject;
+    assignValues(_contact, fillResult.contacts[0]);
```

Comment:
`contacts[0]` - i know you can create multiple contacts with a single create form (eg. district and primary contact). can contact-edit forms when multiple contacts at once?

Reply:
We only edit the subject contact